### PR TITLE
Feat: Log out functionality

### PR DESCRIPTION
**Pull request by Matsuuu**

### CODE REVIEW

File: api/pkg/auth/auth.go [hunk 1]
```diff
@@ -46,10 +46,15 @@ func (j *JwtHandler) GenerateToken(c Claims) (string, error) {
 	return signedToken, nil
 }
 
-//InvalidateToken is used to sign a user out by setting the exp time to 0
-func (j *JwtHandler) InvalidateToken(token string) string {
-
-	return ""
+//LogOut is used to sign a user out by setting the exp time to 0
+func LogOut(writer http.ResponseWriter, r *http.Request ) {
```
Comment (Sam-Lane) [inline api/pkg/auth/auth.go:50]: Should we keep the function name as it was previously? As it is strictly just invaildating the token.

Comment (Matsuuu) [reply api/pkg/auth/auth.go:50]: As it's invaldiating the cookie, and not the JWT token per se, I would say let's name it something like "InvalidateCookie" or "InvalidateSession"

Comment (Sam-Lane) [reply api/pkg/auth/auth.go:50]: JWT's aren't cookies or session though

Comment (Matsuuu) [reply api/pkg/auth/auth.go:50]: Yeah but the function sets the cookie's date in the past, invalidating the cookie

**Pull request merged**